File: problem-10/problem-10.go
package main

import (
	"fmt"

	"github.com/MPadilla198/utils"
)

func main() {
	const SIZE = 2000000

	// Create prime sieve
	prime_sieve := utils.NewPrimeSieve(SIZE)

	sum := 0

	// Iterate sieve, adding all primes
	for i, val := range prime_sieve {
		if i < 2 {
			continue
		}

		if val {
			sum += i
		}
	}

	fmt.Println(sum)
}

File: problem-10/problem-10.ts
import { NewPrimeSieve } from './../utils/typescript/prime_sieve';


const SIZE = 2000000

// Create prime sieve
const prime_sieve: boolean[] = NewPrimeSieve(SIZE)

let i = 0
let sum = 0
for (const val of prime_sieve) {
    if (val) {
        sum += i
    }
    i++
}

console.log(sum)